### PR TITLE
Test workflow runs

### PR DESCRIPTION
**Pull request by it-ony**
# Problem

Orchestration is not tested

# Solution

Add tests 

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [ ] Has the CHANGELOG been updated?
- [ ] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the TESTING_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

File: test/webtest/src/test/java/com/onfido/qa/websdk/mock/config/UploadDocumentConfig.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import com.onfido.qa.websdk.DocumentType;
+
+import java.util.List;
+
+@SuppressWarnings("PublicField")
+public class UploadDocumentConfig extends ConfigBase {
+
+    public static final int DEFAULT_TIMEOUT = 1209600;
+
+    @JsonProperty("ngc_enabled")
```
Comment (zoeradkani) [inline test/webtest/src/test/java/com/onfido/qa/websdk/mock/config/UploadDocumentConfig.java:13]:
```suggestion
    @JsonProperty("nfc_enabled")
```

File: test/webtest/src/test/java/com/onfido/qa/websdk/test/WorkflowIT.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
+package com.onfido.qa.websdk.test;
+
+import com.onfido.qa.websdk.mock.TaskDefinition;
+import com.onfido.qa.websdk.mock.WorkflowRun;
+import com.onfido.qa.websdk.page.BasePage;
+import com.onfido.qa.websdk.page.CountrySelector;
```
Comment (zoeradkani) [inline test/webtest/src/test/java/com/onfido/qa/websdk/test/WorkflowIT.java:6]: we have combine the CountrySelector and DocumentTypeSelector, this test should actually fail. use RestrictedDocumentSelection instead.

File: test/webtest/src/test/java/com/onfido/qa/websdk/test/WorkflowIT.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+public class WorkflowIT extends WebSdkIT {
+
+    static UUID workflowRunId = UUID.randomUUID();
+
+    @DataProvider
+    public static Object[][] foo() {
+
+        return new Object[][]{
+                {TaskDefinition.UPLOAD_DOCUMENT, CountrySelector.class},
```
Comment (zoeradkani) [inline test/webtest/src/test/java/com/onfido/qa/websdk/test/WorkflowIT.java:29]: should be RestrictedDocumentSelection.class

File: test/webtest/src/test/java/com/onfido/qa/websdk/test/WorkflowIT.java [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+                {TaskDefinition.UPLOAD_DOCUMENT_PHOTO, CountrySelector.class},
```
Comment (zoeradkani) [inline test/webtest/src/test/java/com/onfido/qa/websdk/test/WorkflowIT.java:30]: same as above

File: test/webtest/src/test/java/com/onfido/qa/websdk/test/WorkflowIT.java (hunk 2 above)
Comment (it-ony) [reply test/webtest/src/test/java/com/onfido/qa/websdk/test/WorkflowIT.java:6]: Won't fail, if the element that it expects to find is present. Which is the case. I'll anyway update it to RestrictedDocumentSelection

**Review by zoeradkani — APPROVED** [review]

**Pull request merged**